system-test/spanner.ts

```diff
@@ -3743,14 +3697,6 @@ describe('Spanner', () => {
       }
     };
 
-    it('GOOGLE_STANDARD_SQL should delete backup', async () => {
-      await deleteBackup(googleSqlBackup2);
-    });
-
-    it.skip('POSTGRESQL should delete backup', async () => {
-      await deleteBackup(postgreSqlBackup2);
-    });
-
```

Comment:
instead of deleting, we can skip pushing backups in RESOURCES_TO_CLEAN and enable this test?

Reply:
no, we can't do that as these backups are in use by restore operations and databases created with restore operations, so we would have to delete those databases first